src/server/ua_server_binary.c

```diff
@@ -618,6 +646,9 @@ createServerSecureChannel(UA_BinaryProtocolManager *bpm, UA_ConnectionManager *c
     if(scs->currentChannelCount >= config->maxSecureChannels &&
        !purgeFirstChannelWithoutSession(bpm))
         return UA_STATUSCODE_BADOUTOFMEMORY;
+    if(config->maxConnections && scs->currentChannelCount >= config->maxConnections &&
+       !purgeFirstConnectionWithoutChannel(cm, bpm))
+        return UA_STATUSCODE_BADOUTOFMEMORY;
```

Comment:
I think there is a dedicated status for that.
Something like out-of-networking-resources or similar.

Reply:
Is it t UA_STATUSCODE_BADTCPNOTENOUGHRESOURCES?
or maybe the more general UA_STATUSCODE_BADRESOURCEUNAVAILABLE?
Those were the only relevant-looking statuscodes I could find.